sunpy/instr/aia.py

```diff
@@ -7,10 +7,14 @@
 import astropy.units as u
 
 from sunpy.map.sources.sdo import AIAMap, HMIMap
+from sunpy.util.decorators import deprecated
 
 __all__ = ['aiaprep']
 
 
+@deprecated("2.0", message="For converting AIA images to level 1.5, please \
+use the `update_pointing` and `register` functions in aiapy: \
+https://gitlab.com/LMSAL_HUB/aia_hub/aiapy")
```

Comment:
Do we need to say that this will be removed in future versions of sunpy?

Reply:
That gets automatically appended to the end of this message, with the removal version automatically calculated from the `since` version, 2.0 in this case. The output looks like,

```python
>>> m_prep = aiaprep(m)
WARNING: SunpyDeprecationWarning: For converting AIA images to level 1.5, please use the `update_pointing` and `register` functions in aiapy: https://gitlab.com/LMSAL_HUB/aia_hub/aiapy This is scheduled for removal in 3.1. [sunpy.util.decorators]
WARNING: SunpyUserWarning: Input data has been cast to float64. [sunpy.image.transform]
```